presto-product-tests/src/main/java/io/prestosql/tests/iceberg/TestIcebergCreateTable.java

```diff
@@ -22,52 +24,63 @@
 import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
 import static io.prestosql.tests.TestGroups.ICEBERG;
 import static io.prestosql.tests.TestGroups.STORAGE_FORMATS;
+import static io.prestosql.tests.hive.util.TemporaryHiveTable.randomTableSuffix;
 import static io.prestosql.tests.utils.QueryExecutors.onPresto;
 
 public class TestIcebergCreateTable
         extends ProductTest
 {
+    @BeforeTestWithContext
+    public void setUp()
+    {
+        onPresto().executeQuery("CREATE SCHEMA iceberg.iceberg");
```

Comment:
CREATE SCHEMA IF NOT EXISTS

Reply:
pre-existing and i'd want to consider this out of scope for this PR. can i?

Reply:
discussed offline